c7n/mu.py

```diff
@@ -644,6 +643,10 @@ def role(self):
         return self.policy.data['mode'].get('role', '')
 
     @property
+    def runtime(self):
+        return self.policy.data['mode'].get('runtime', DEFAULT_RUNTIME)
```

Comment:
that seems odd.. default runtime returns a dictionary? and explicit config returns a string?